generate_geojson.sh

```diff
@@ -11,7 +11,7 @@ if [[ ${FILE_PARAM} == http* ]]; then
   scif run betydb2geojson --betydb_url "${FILE_PARAM}" "${DESTINATION_FILE}"
 elif [[ ${FILE_PARAM} == *.shp ]]; then
   scif run shp2geojson "${FILE_PARAM}" "${DESTINATION_FILE}"
-elif [[ ${FILE_PARAM} == *.json || ${FILE_PARAM} == *.geojson ]]; then
+elif [[ ${FILE_PARAM} == *.json ]]; then
```

Comment:
```suggestion
elif [[ ${FILE_PARAM} == *.json || ${FILE_PARAM} == *.geojson ]]; then
```
you need to add this change back in

Reply:
I must have forgotten to merge locally with develop; I just did that. I'll make sure I don't forget that much anymore